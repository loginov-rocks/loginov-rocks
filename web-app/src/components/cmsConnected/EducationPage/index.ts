import { EducationPage } from './EducationPage';

export { EducationPage };
