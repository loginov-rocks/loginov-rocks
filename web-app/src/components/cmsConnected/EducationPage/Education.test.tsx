/* eslint-disable import/no-extraneous-dependencies */

import { shallow } from 'enzyme';
import * as React from 'react';

import { EducationPage } from './EducationPage';

it('matches snapshot', () => {
  const wrapper = shallow(<EducationPage render={() => null} sections={[]} title="Education" />);

  expect(wrapper).toMatchSnapshot();
});
