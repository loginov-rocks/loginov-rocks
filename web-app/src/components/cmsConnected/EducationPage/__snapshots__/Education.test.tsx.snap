// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<Fragment>
  <h1>
    Education
  </h1>
</Fragment>
`;
